From 0e05d4116678de3a7d913177a8fc63f10e58ad24 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sat, 18 Jan 2020 00:00:21 +0100
Subject: [PATCH] configure.ac: remove po/Makefile.in

The file po/Makefile.in is automatically added to AC_OUTPUT while using
gettexize

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: not upstreamable]
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 3876adb..221dacf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -471,7 +471,6 @@ LDFLAGS="$TEMP_LDFLAGS"
 AC_CONFIG_FILES([
 	Makefile
 	src/Makefile
-  po/Makefile.in
   man/Makefile
   data/camera1-dist.conf
   data/camera2-dist.conf
-- 
2.24.1

